src/main/java/com/onfleet/demo/homework/collection/Tasklist.java

```diff
@@ -135,8 +132,7 @@ public static double calculateDistanceForPoints(final Task start, final Task fin
    * tasks for this {@link Driver}.
    */
   private void recalculateLoadEstimate() {
-    final double taskCountAsDouble = (double)this.taskCount;
-    this.loadEstimate += ((this.knownDistance / taskCountAsDouble) * (taskCountAsDouble * TASK_COUNT_WEIGHT));
+    this.loadEstimate = this.knownDistance;
```

Comment:
for now, eliminating any notion of "weight" when calculating load. it's just the distance between all tasks.